diff --git a/src/unix/internal.h b/src/unix/internal.h
index 72d8da8a..378b0b24 100644
--- a/src/unix/internal.h
+++ b/src/unix/internal.h
@@ -60,14 +60,6 @@
 # include <AvailabilityMacros.h>
 #endif
 
-#if defined(__ANDROID__)
-int uv__pthread_sigmask(int how, const sigset_t* set, sigset_t* oset);
-# ifdef pthread_sigmask
-# undef pthread_sigmask
-# endif
-# define pthread_sigmask(how, set, oldset) uv__pthread_sigmask(how, set, oldset)
-#endif
-
 #define ACCESS_ONCE(type, var)                                                \
   (*(volatile type*) &(var))
 
